### ISSUE TITLE
Unknown command: wire_expression2_concmd

### ISSUE THREAD

**Issue by deathking61** [wiremod/wire#1470]
This command doesn't work i have tried to check the box to enable and disable it repeatedly and all i get in the console is: Unknown command: wire_expression2_concmd so I have tried to workshop one i have tried the gihub one i also removed all cache files to prevent any and all issues but it still persists! any way to fix this? as some of my e2's won't work without this enabled and yes i have tried RCON too as sometimes that fixes it but this doesn't help so yea any way to actually get this to work?

**Comment by GrimboNutter** [comment]
Did you ever find a fix?

**Comment by viral32111** [comment]
This seriously needs to be fixed asap.

**Comment by Nebual** [comment]
Works for me.
1. Do you see any errors in console that seem remotely wire-related? If you have any other E2 extensions that have clientside components, which are failing to load, that might be cascading to cl_console.lua too
2. Do you have the convar `wire_expression2_print_max`? (its setup in cl_debug.lua so curious if any of the cl_ files are loading for you)

**Comment by bigdogmat** [comment]
I tried looking into this after a friend pointed me towards an addon that caused it. His issue was caused by the `TFA Base [ Reduxed ]` addon. After installing it the command did indeed go away, and the reason being is for whatever reason `OnEntityCreated` was never being called with the `LocalPlayer`. I looked into the addon and didn't see anything that would've caused this so I'm not sure.

The hook that creates these commands should probably be changed to `Initialize`, I don't see any reason to be using `OnEntityCreated`.

@deathking61 If you have this addon installed, does removing it fix the issue?
